samples/unity-of-bugs/Assets/Resources/Sentry/SentryOptions.asset

```diff
@@ -25,6 +25,7 @@ MonoBehaviour:
   <AttachScreenshot>k__BackingField: 1
   <ScreenshotQuality>k__BackingField: 2
   <ScreenshotCompression>k__BackingField: 75
+  <AddLogsAsBreadcrumbs>k__BackingField: 1
```

Comment:
Doesn't seem a correct name? Some older version of the file? Or where does this come from?